data/events/2020-poznan.yml

```
@@ -100,10 +100,13 @@ team_members: # Name is the only required field for team members.
     employer: "Mirantis"
     twitter: "eftepede"
     image: bartek-stalewski.jpg
-  - name: "Klaudia Jasniak"
+  - name: "Klaudia Nawrot"
     employer: "JCommerce"
-    linkedin: "https://www.linkedin.com/in/klaudia-ja%C5%9Bniak-97283180/"
+    linkedin: "https://www.linkedin.com/in/klaudia-nawrot-ja%C5%9Bniak-97283180/"
     image: klaudia-jasniak.jpg
```

I assume this should be changed to `klaudia-nawrot.jpg` ?

```suggestion
    image: klaudia-nawrot.jpg
```